integration-test/src/test/java/org/ehcache/integration/statistics/TierCalculationTest.java

```diff
@@ -68,6 +74,7 @@ public void before() throws Exception {
       .withCache("cache", cacheConfiguration)
       .using(new DefaultPersistenceConfiguration(diskPath.newFolder()))
       .using(statisticsService)
+      .using(new DefaultTimeSourceService(timeSource))
```

Comment:
Just use an anonymous inner `TimeSourceService` to hand off the  `TestTimeSource` and then you do not need to add the injection constructor on the default one.